frontend/admin-client/src/app/bookings/booking/booking.component.html

```diff
@@ -2,11 +2,11 @@
   <!--  booking information-->
   <div class="d-flex">
     <div class="font-weight-bold">BOOKING ID: {{booking?.id}}</div>
-    <div class="ml-4 font-weight-bold">
-      {{booking?.flights[0]?.route?.origin?.city}} - {{booking?.flights.slice(-1)[0]?.route?.destination?.city}}
+    <div class="ml-4 font-weight-bold" *ngIf="booking.flights && booking.flights.length > 0">
```

Comment:
this could be simplified to `bookings.flights?.length > 0`

Reply:
You are right